### DISCUSSION TITLE
Create function equals ".function()"

### DISCUSSION THREAD

**Discussion by Thomas-Teo** [codewars/codewars.com#2717] [Q&A]
How i create a function equals `str.toUpperCase()` (string+dot+function)?
create methods and functions I know, but I wanted one like `str.toUpperCase()`, so I just use the dot and the function and it works

**Comment by isaacmcdougal** [comment]
I'm guessing this is for Javascript. You could probably google this, but [this article](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Basics) and [this article](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object_prototypes) are good places to start.

**Reply by Thomas-Teo** [reply]
I don't know how to search this

**Reply by isaacmcdougal** [reply]
Did you read the articles I linked? This is all object oriented programming in Javascript. If you want to add methods to an already existing class (like String), you need to utilize the prototype chain. Both of the articles I've linked in my earlier reply should help you.

**Reply by Thomas-Teo** [reply]
Sorry, I didn't see the links.
